### PR TITLE
command: help text for generate-config-out

### PR DESCRIPTION
**Pull request by kmoe**
Add help text for `terraform plan -generate-config-out=path`. Reflow the Other Options section to make it fit.

### CODE REVIEW

**Review by liamcervante — APPROVED** [review]

File: internal/command/plan.go [hunk 1]
```diff
@@ -247,33 +247,42 @@ Plan Customization Options:
...
 
-  -no-color           If specified, output won't contain any color.
+  -lock-timeout=0s           Duration to retry a state lock.
 
-  -out=path           Write a plan file to the given path. This can be used as
-                      input to the "apply" command.
+  -no-color                  If specified, output won't contain any color.
 
-  -parallelism=n      Limit the number of concurrent operations. Defaults to 10.
+  -out=path                  Write a plan file to the given path. This can be
```
Comment (rkoron007) [inline internal/command/plan.go:277]: Tiny wording tweaks for active voice, feel free to ignore if you don't like!
```suggestion
  -out=path                  Write a plan file to the given path. You can use
```

File: internal/command/plan.go [hunk 2]
```diff
@@ -247,33 +247,42 @@ Plan Customization Options:
...
+                             used as input to the "apply" command.
```
Comment (rkoron007) [inline internal/command/plan.go:278]: Tiny wording tweaks for active voice, feel free to ignore if you don't like!
```suggestion
                             this as input for the "apply" command.
```

File: internal/command/plan.go [hunk 3]
```diff
@@ -247,33 +247,42 @@ Plan Customization Options:
...
-                      2 - Succeeded, there is a diff
+  -detailed-exitcode         Return detailed exit codes when the command exits.
+                             This will change the meaning of exit codes to:
+                             0 - Succeeded, diff is empty (no changes)
+                             1 - Errored
+                             2 - Succeeded, there is a diff
 
-  -input=true         Ask for input for variables if not directly set.
+  -generate-config-out=path  (Experimental) If import blocks are present in
+                             config, instructs Terraform to generate HCL for any
```
Comment (rkoron007) [inline internal/command/plan.go:261]: I noticed in this file we spell out "configuration" so adding a suggestion:
```suggestion
                             the current configuration, this command instructs Terraform to generate HCL for any
```

File: internal/command/plan.go [hunk 4]
```diff
@@ -247,33 +247,42 @@ Plan Customization Options:
...
+                             imported resource not already present. The config
```
Comment (rkoron007) [inline internal/command/plan.go:262]:
```suggestion
                             imported resources not already present. Your configuration 
```

File: internal/command/plan.go [hunk 5]
```diff
@@ -247,33 +247,42 @@ Plan Customization Options:
...
+                             will be written to a file at PATH, which must not
```
Comment (rkoron007) [inline internal/command/plan.go:263]:
```suggestion
                             is written to a new file at PATH, which must not
```

**Review by rkoron007 — COMMENTED** [review]
Adding tiny suggestions for style, but otherwise looks good to me!

**Comment by kmoe** [comment]
Thank you very much, @rkoron007! I've taken some of your suggestions on board, still keeping it brief.

File: internal/command/plan.go (hunk 1 above)
Comment (kmoe) [reply internal/command/plan.go:277]: I agree with these. Would you mind opening a separate PR if you have time? This PR should only add the `-generate-config-out` text.

**Pull request merged**